testapp-flutter/lib/game.dart

```diff
@@ -182,13 +187,17 @@ class _GameState extends State<Game> {
       buffer.writeln("Recovered ${character!.curHp - lastHp} HP!");
     }
     setState(() {
+      Spinner.disable(context);
```

Comment:
i am working on exposing `ShowPylonsAnimation` so you can replace this in future.